packages/statemanager/src/cache/types.ts

```diff
@@ -1,6 +1,58 @@
-import type { CacheType } from '@ethereumjs/common'
+export enum CacheType {
+  LRU = 'lru',
+  ORDERED_MAP = 'ordered_map',
+}
 
 export interface CacheOpts {
   size: number
   type: CacheType
 }
+
+export type CacheSettings = {
+  deactivate: boolean
+  type: CacheType
+  size: number
+}
+
+export interface CacheStateManagerOpts {
```

Comment:
Maybe drop the `StateManager` here from the name

Reply:
If we remove StateManager, we'll have a naming conflict with `CacheOpts` declared above. I am not 100% satisfied with the general naming, and there is some overlap between these interfaces/types (which makes me want to unify them somehow), but there does not seem to be sufficient overlap to really unify them. Open to suggestions!

Reply:
Maybe give this a second thought, but I would kill off the `deactivate` part and take the `size === 0` as the one and only way to deactivate and the remove the `CacheOpts` and only use the `CachesStateManagerOpts` (then renamed to `CacheOpts` 😋) as the one thing for all.

Reply:
Interesting, just seeing this comment I'll give that a try, would really like to simplify further